simple_qa_rnn/.gitignore

```diff
@@ -1,4 +1,8 @@
-data/
+data/*
```

Comment:
This is not a big deal. They can be easily switched using the arguments "--data_cache" and "--vector_cache". The person running the program needs to point to his/her GloVe directory